glean-core/rlb/src/dispatcher/global.rs

```diff
@@ -63,9 +63,19 @@ pub fn flush_init() -> Result<(), DispatchError> {
 ///
 /// This will initiate a shutdown of the worker thread
 /// and no new tasks will be processed after this.
-/// It will not block on the worker thread.
 pub fn try_shutdown() -> Result<(), DispatchError> {
```

Comment:
Should this be renamed to `shutdown`? We're not only trying anymore, we're actively waiting for it to shutdown.